src/cfdcore_schnorrsig.cpp

```diff
@@ -312,6 +314,54 @@ Pubkey SchnorrUtil::ComputeSigPoint(
   return ConvertSecpPubkey(secp_sigpoint);
 }
 
+Pubkey SchnorrUtil::ComputeSigPointBatch(
+    const std::vector<ByteData256> &msg,
+    const std::vector<SchnorrPubkey> &nonces, const SchnorrPubkey &pubkey) {
+  auto ctx = wally_get_secp_context();
+
+  if (msg.size() != nonces.size() || msg.size() < 1) {
```

Comment:
I think it is better to use the empty function.
```suggestion
  if (msg.size() != nonces.size() || msg.empty()) {
```

Reply:
Thanks indeed better. Fixed